usersimcrs/items/item_collection.py

```diff
@@ -113,5 +113,12 @@ def get_possible_property_values(self, property: str) -> List[str]:
         Returns:
             List of possible values.
         """
-        # TODO: https://github.com/iai-group/UserSimCRS/issues/112
-        return []
+        values: List[Any] = []
```

Comment:
Wouldn't it be more efficient to make it a set?

Reply:
I was getting some error in another part of the code when I used a set. However, I found a workaround with a set.